Sources/SwiftFormat/API/Configuration.swift

```diff
@@ -34,6 +34,7 @@ public struct Configuration: Codable, Equatable {
     case lineBreakBeforeControlFlowKeywords
     case lineBreakBeforeEachArgument
     case lineBreakBeforeEachGenericRequirement
+    case lineBreakBetweenAttributes
```

Comment:
Can we update the name of this to be `lineBreakBetweenDeclarationAttributes` to match the new behavior?